README.md

```diff
@@ -107,6 +107,8 @@ Options:
   -j, --parallel <number>     Run tests in parallel
   -J, --autoParallel          Run tests in parallel (automatically detect core count)
   --tmpDir <path>             Directory to test modules in
+  --withTags <tag>            Only test modules from the lookup that contain a matching tag field
+  --excludeTags <tag>         Specify which tags to skip from the lookup
```

Comment:
Just realised that Myles said this earlier in https://github.com/nodejs/citgm/pull/308#pullrequestreview-17039480. 

>we already have tests for include/exclude: https://github.com/gdams/citgm/blob/f87bb014327be7322d4c5efe5a22a2e5966b5bc3/test/bin/test-citgm-all.js#L119-L139

That doesn't cover multiple tags.